package.json

```diff
@@ -18,7 +18,7 @@
     "start": "yarn start:module",
     "start:module": "webpack-dev-server --config webpack.examples.config.js",
     "start:script": "http-server ./",
-    "build": "rm -rf lib && webpack && echo 'Object.assign(THREE,__three_vrm__);' >> lib/index.js && tsc --emitDeclarationOnly",
```

Comment:
This code does not work in Windows environment, so remove it and use assign.ts as the entry point of index.js to achieve the same result.